package/Editor/Models/Models.cs

```diff
@@ -209,19 +319,75 @@ private static void FetchAllPublicTextures()
             }
         }
 
-        private static async Task FetchAllPrivateModels()
+        /// <summary>
+        /// Processing to get all quickstart models
+        /// </summary>
+        /// <returns></returns>
+        private static async Task FetchAllQuickStartModels()
         {
             if (!isProcessing)
             {
+                while (true)
```

Comment:
Why is there a while(true) here ? seems weird / useless

Reply:
So without this while, is fetching only one page of model.
But I'm agree, and may be we can use thoses lines to:
```
 PaginationTokenPublic = "";
 Debug.Log("no next page to fetch.");
```
To stop the process. I'll check this

Reply:
Fix it !

With this fix we also fix another bug, which it was, sometines on redraw or reopening, first models or the first page were in double.